### ISSUE TITLE
From system log: Updating state for binary_sensor.XXXXX (<class 'custom_components.reolink_dev.binary_sensor.MotionSensor'>) took 0.408 seconds. Please report it to the custom integration author.

### ISSUE THREAD

**Issue by troelde** [fwestenberg/reolink_dev#546]
Logger: homeassistant.helpers.entity
Source: helpers/entity.py:532
First occurred: 04:12:11 (1 occurrences)
Last logged: 04:12:11

Updating state for binary_sensor.inngangsparti_motion (<class 'custom_components.reolink_dev.binary_sensor.MotionSensor'>) took 0.408 seconds. Please report it to the custom integration author.

**Comment by mnpg** [comment]
Hi @troelde, from your log, it's seems that the nginx server of your camera has problem (Bad Gateway). Reboot your camera and check if it solutes your problem. Sometimes, my cameras do not responded normally after fewer days of continious usage. I reboot them regulary to avoid these problems.

**Comment by troelde** [comment]
Thank you. I rebooted camera last evening and most of the log entries disappeared.
Now, only this error is logged every second minute. That seem to be related to Reolink. PS! I have only one camera installed.

`ERROR (MainThread) [async_upnp_client.ssdp] Received error: [Errno 101] Network unreachable, transport: <_SelectorDatagramTransport fd=17 read=polling write=<idle, bufsize=0>>`

Any ideas?

**Comment by mnpg** [comment]
Not sure that your last problem depends on the reolink_dev plugin.
Take a look at this issue at the home-assistant/core github webpage : https://github.com/home-assistant/core/issues/73988

**Comment by troelde** [comment]
Yes, you are right. Thank you. I will follow up issue in https://github.com/home-assistant/core/issues/73988.

However, got a few new error log entries since last time:
[22020813_ha.log](https://github.com/fwestenberg/reolink_dev/files/9331672/22020813_ha.log)

**Comment by mnpg** [comment]
From your log, it's seems that you can't access to the https port of your camera (port closed?) :` Cannot connect to host 192.168.1.103:443`
Check if you can open the web interface of your cam via your webbrowser: https://192.168.1.103
If not, verify if the https port is open in the reolink app (Network > Advanced > Port Settings).

**Comment by troelde** [comment]
> Check if you can open the web interface of your cam via your webbrowser: https://192.168.1.103/
It works. Of course I get an certificate error in browser because certificate is self signed by reo-link, but that is as expected. I can continue by ignoring the warning in browser.
